### PR TITLE
SeaDrop integration process

### PR DESCRIPTION
**Pull request by ryley-o**
Add docs that provide instructions for using our SeaDrop integration.

closes https://linear.app/art-blocks/issue/PLT-644/publish-seadrop-integration-docs

### CODE REVIEW

File: minter-suite/seadrop-integration.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+order: -20
+---
+
+# SeaDrop Integration
+
+This page provides an overview of an integration solution for the Art Blocks Shared Minter Suite with SeaDrop. This is a special integration that allows Art Blocks projects to be minted and sold on OpenSea via SeaDrop, and is not a standard integration with the Shared Minter Suite.
```
Comment (aaronpenne) [inline minter-suite/seadrop-integration.md:7]: change `with OpenSea` to `with OpenSea's SeaDrop <contract/infra/system>`

File: minter-suite/seadrop-integration.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+---
+
+## Integration Process
+
+The integration process follows the custom, one-off minter steps, and supports minting on any project that uses the Shared Minter Suite.
+
+### 1. Artist configures their project
+
+The artist configures their project as they normally would, using the Art Blocks Creator Dashboard.
```
Comment (aaronpenne) [inline minter-suite/seadrop-integration.md:19]: add a link to docs `https://docs.artblocks.io/`

File: minter-suite/seadrop-integration.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+Following the custom, one-off minter steps, the admin must approve the minter for the project's core contract.
+
+Engine Admin calls `approveMinterForContract` on the shared minter filter contract (via e.g. etherscan or gnosis safe tx builder), passing in the address of their core contract, and the `SeaDropXArtBlocksShim` custom minter contract
+
+### 3. Artist configures the custom minter for their project
+
+Following the custom, one-off minter steps, the artist will need to configure the custom minter for their project. The artist will need to configure the custom minter via etherscan, etc. Note that the typical minter filter function for setting a minter for a project, `setMinterForProject` should be used by the artist or admin to assign the custom minter for the project.
+
+### 4. Artist configures their drop on OpenSea's Studio
```
Comment (aaronpenne) [inline minter-suite/seadrop-integration.md:50]: couldn't find a great entry point into OS Studio docs, maybe `https://docs.opensea.io/docs/part-2-edit-collection-settings` but would be worth checking

File: minter-suite/seadrop-integration.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+The artist will need to configure their drop on OpenSea's Studio.
+
+In OpenSea Studio, the artist should see a new contract available to edit, which will be the `SeaDropXArtBlocksShim` contract. The artist should configure the drop as they normally would, using the OpenSea Studio interface.
+
+A few options that don't make sense for Art Blocks projects, such as IPFS metadata (Art Blocks projects are on-chain), should be left blank.
+
+If SeaDrop is configured to sell less tokens than the project's max invocations, minting on OpenSea will be limited to whatever is configured in SeaDrop.
+
+!!!warning
```
Comment (aaronpenne) [inline minter-suite/seadrop-integration.md:60]: good call out

**Review by aaronpenne — APPROVED** [review]

File: minter-suite/seadrop-integration.md (hunk 1 above)
Comment (ryley-o) [reply minter-suite/seadrop-integration.md]: updated in [142c186](https://github.com/ArtBlocks/artblocks-docs/pull/314/commits/142c1866a6bf1f6cad6189f26aef34be0023f256)

File: minter-suite/seadrop-integration.md (hunk 2 above)
Comment (ryley-o) [reply minter-suite/seadrop-integration.md]: assuming you meant `https://create.artblocks.io` - updated in [142c186](https://github.com/ArtBlocks/artblocks-docs/pull/314/commits/142c1866a6bf1f6cad6189f26aef34be0023f256)

File: minter-suite/seadrop-integration.md (hunk 3 above)
Comment (ryley-o) [reply minter-suite/seadrop-integration.md:50]: good call, and I agree with the suggested link!
updated in [5be7506](https://github.com/ArtBlocks/artblocks-docs/pull/314/commits/5be7506abb5ff27d5a052a4315eb57dd96d5adcb)

**Pull request merged**